client/warnings.go

```diff
@@ -1,82 +0,0 @@
-// Copyright (c) 2014-2020 Canonical Ltd
```

Comment:
Why is `internals/cli/cmd_warnings` kept, but `client/warnings` removed? I would assume that CLI and client have the same set of commands/APIs.

Reply:
The `pebble warnings` command lists warnings, but because warnings are now implemented in terms of notices, there's no warnings API or state anymore -- it's all done via notices.